agent/tool-scripts/meta.json

```diff
@@ -37,7 +37,8 @@
 		"turbostat": null,
 		"user-tool": null,
 		"virsh-migrate": null,
-		"vmstat": null
+		"vmstat": null,
+		"pcp-transient": null
```

Comment:
This is a naming problem we have to handle when we do this, and I am not sure exactly how to deal with this.

Perhaps we use the same name, `pcp`, but we default to persistent, and only if the user registers the a new switch, perhaps `--transient`, we use the transient tool.  We'd also have a `--persistent` register tool option as well.

Reply:
This may be worth discussing tomorrow morning, as it depends on how many other tools will have both persistent and transient options in the future, and may also require a change to tool metadata structure to future-proof implementation.

Reply:
I've made tentative changes to naming externally, though I'm not sure whether this is the right call long-term. There are trade-offs to an external implementation vs. an internal rework of persistent vs. transient tools, tool metadata, and how they are handled in TDS/TM, so it may be worth weighing the pros and cons of each.